hardhat.config.ts

```diff
@@ -56,10 +56,21 @@ const gasOption =
 
 const config: HardhatUserConfig = {
   solidity: {
-    version: "0.6.10",
-    settings: {
-      optimizer: { enabled: true, runs: 200 },
-    },
+    compilers: [
+      {
+        version: "0.6.10",
+        settings: { optimizer: { enabled: true, runs: 200 } },
+      },
+      {
+        version: "0.8.11",
```

Comment:
We using 0.8.11?

Reply:
This was for when I had to import some contracts from Notional and deploy them for testing. (Before they were tested).

I can probably remove this now since I'm just testing against their prod deployments.
good catch

Reply:
Removed it.